client/src/modules/debtors/groups.update.html

```diff
@@ -129,50 +130,50 @@
         <div class="panel panel-default">
           <div class="panel-heading">
             <span class="fa fa-book" aria-hidden="true"></span>
-            {{ "DEBTOR_GROUP.POLICIES.TITLE" | translate }}
+            <span translate>DEBTOR_GROUP.POLICIES.TITLE</span>
           </div>
           <div class="panel panel-body">
 
             <div class="row">
               <div class="col-md-12">
-                <h5><b>{{ "DEBTOR_GROUP.POLICIES.SUBSIDIES.TITLE" | translate }}</b></h5>
+                <h5><b translate> DEBTOR_GROUP.POLICIES.SUBSIDIES.TITLE </b></h5>
 
                 <!-- Only show if the subsidies option is unchecked -->
                 <p ng-if="GroupUpdateCtrl.group.apply_subsidies===false" class="text-info">
-                  <span class="glyphicon glyphicon-info-sign"></span> {{ "DEBTOR_GROUP.POLICIES.SUBSIDIES.INFO" | translate }}
+                  <span class="glyphicon glyphicon-info-sign"></span> <span translate>DEBTOR_GROUP.POLICIES.SUBSIDIES.INFO</span>
                 </p>
                 <div class="checkbox">
                   <label>
                     <input ng-model="GroupUpdateCtrl.group.apply_subsidies" name="apply_subsidies" type="checkbox">
-                    {{"DEBTOR_GROUP.POLICIES.SUBSIDIES.LABEL" | translate }}
+                    <span translate>DEBTOR_GROUP.POLICIES.SUBSIDIES.LABEL</span>
                   </label>
                 </div>
               </div>
             </div>
             <div class="row">
               <div class="col-md-12">
-                <h5><b>{{"DEBTOR_GROUP.POLICIES.DISCOUNTS.TITLE" | translate }}</b></h5>
+                <h5><b translate>DEBTOR_GROUP.POLICIES.DISCOUNTS.TITLE</b></h5>
                 <p ng-if="GroupUpdateCtrl.group.apply_discounts===false" class="text-info">
-                  <span class="fa fa-info-circle" aria-hidden="true"></span> {{"DEBTOR_GROUP.POLICIES.DISCOUNTS.INFO" | translate }}
+                  <span class="fa fa-info-circle" aria-hidden="true"></span><span translate>DEBTOR_GROUP.POLICIES.DISCOUNTS.INFO</span>
                 </p>
                 <div class="checkbox">
                   <label>
                     <input ng-model="GroupUpdateCtrl.group.apply_discounts" name="apply_discounts" type="checkbox">
-                    {{"DEBTOR_GROUP.POLICIES.DISCOUNTS.LABEL" | translate }}
+                    <span translate>{{"DEBTOR_GROUP.POLICIES.DISCOUNTS.LABEL</span>
```

Comment:
This is broken - it should be `<span translate>DEBTOR_GROUP.POLICIES.DISCOUNTS.LABEL</span>`.